at_secondary/at_persistence_secondary_server/lib/src/keystore/hive_keystore.dart

```diff
@@ -381,4 +378,26 @@ class HiveKeystore implements SecondaryKeyStore<String, AtData?, AtMetaData?> {
       await persistenceManager.openVault(persistenceManager.atsign!);
     }
   }
+
+  bool _isExpired(key) {
+    if (_metaDataCache[key]?.expiresAt == null) {
+      return false;
+    }
+    return _metaDataCache[key]!.expiresAt!.isBefore(DateTime.now().toUtc());
+  }
+
+  bool _isBorn(key) {
```

Comment:
Actually, the right method name should _isNotBorn. If "availableAt" is before "DateTime.now().toUtc()" then it is not born. Born is when availableAt is after now.